spec/system/admin/searchgov_domain_management_spec.rb

```diff
@@ -1,21 +1,21 @@
 # frozen_string_literal: true
 
-describe 'Searchgov Domain Management' do
+describe 'Searchgov Domain' do
```

Comment:
Let's use the page title: `describe 'Search.gov Domains'`. The filename should be tweaked accordingly, to `spec/system/admin/searchgov_domains_spec.rb`

Reply:
Changed.